semeio/communication/reporter.py

```diff
@@ -10,9 +11,17 @@ def __init__(self, output_dir):
             err_fmt = "Expected output_dir to be an absolute path, received '{}'"
             raise ValueError(err_fmt.format(output_dir))
 
+    def publish_csv(self, namespace, data):
+        output_file = self._prepare_output_file(namespace) + ".csv"
+        if isinstance(data, pd.DataFrame):
```

Comment:
The typical Python approach here is to support publishing as csv if it has a `to_csv`-method. Duck typing 🦆

Reply:
🦆